packages/css-library/src/stylesheets/_override-function.scss

```diff
@@ -0,0 +1,159 @@
+@use 'sass:math';
```

Comment:
I had to make some modifications to this file in order for it to work with `dart-sass` but I expect that this will also eventually go away as we remove stylesheets.